pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -178,8 +179,8 @@ func (exp *flowExporter) initFlowExporter(collector net.Addr) error {
 
 func (exp *flowExporter) sendFlowRecords() error {
 	sendAndUpdateFlowRecord := func(key flowexporter.ConnectionKey, record flowexporter.FlowRecord) error {
-		dataRec := ipfix.NewIPFIXDataRecord(exp.templateID)
-		if err := exp.sendDataRecord(dataRec, record); err != nil {
+		dataSet := ipfix.NewSet(ipfixentities.Data, exp.templateID, false)
```

Comment:
do we need to send a separate set for each record? will there be an issue if we do multiple records in one set?

Reply:
We can combine the records into one set. Updated. Thanks!

Reply:
Hi Yiou, We have a bug with UDP transport with respect to the message size check. There could be an issue if the size of records exceeds the allowed message size in the case of UDP transport. This depends on the number of flows that are exported.

This PR will solve it but will be available only with go-ipfix v0.4.0: https://github.com/vmware/go-ipfix/pull/92

Reply:
Got it. Thanks for solving it. So shall we change back to one set per record for current PR and wait until go-ipfix v0.4.0 to support multiple records per set?

Reply:
Yes, lets do that. Thanks.